Gordon360/Extensions/DateTimeExtensions.cs

```diff
@@ -20,7 +20,49 @@ static public class DateTimeExtensions
     {
         if (nullableDateTime is DateTime dateTime)
             return DateTime.SpecifyKind(dateTime, DateTimeKind.Utc);
+        return null;
+    }
+
+    /// <summary>
+    /// Converts UTC DateTime to Eastern Standard Time 
+    /// List of time zones specified by TimeZoneInfo https://i.stack.imgur.com/zHzGt.png
+    /// </summary>
+    /// <param name="dateTime">DateTime with kind UTC</param>
+    /// <returns>Converted DateTime with kind Unspecified</returns>
+    static public DateTime ConvertUtcToEst(this DateTime dateTime) => TimeZoneInfo.ConvertTimeFromUtc(dateTime, TimeZoneInfo.FindSystemTimeZoneById("Eastern Standard Time"));
+
+
+    /// <summary>
+    /// Converts UTC DateTime? to Eastern Standard Time 
+    /// List of time zones specified by TimeZoneInfo https://i.stack.imgur.com/zHzGt.png
+    /// </summary>
+    /// <param name="nullableDateTime">DateTime? with kind UTC</param>
+    /// <returns>Converted DateTime with kind Unspecified or null if parameter is null</returns>
+    static public DateTime? ConvertUtcToEst(this DateTime? nullableDateTime)
+    {
+        if (nullableDateTime is DateTime dateTime)
+            return TimeZoneInfo.ConvertTimeFromUtc(dateTime, TimeZoneInfo.FindSystemTimeZoneById("Eastern Standard Time"));
```

Comment:
Rather than duplicating the logic in two places, you can just call the non-nullable method here:
```suggestion
        if (nullableDateTime is DateTime dateTime)
            return dateTime.ConvertUtcToEst();
```

Reply:
I was actually thinking about this and actually have this in a commit that I haven't pushed yet.